types/lodash/tslint.json

```diff
@@ -5,6 +5,7 @@
         "max-line-length": false,
         "no-unnecessary-generics": false,
         "typedef-whitespace": false,
-        "unified-signatures": false
+        "unified-signatures": false,
+        "use-default-type-parameter": false
```

Comment:
why is this needed?

Reply:
Please see https://github.com/DefinitelyTyped/DefinitelyTyped/pull/42220#discussion_r376771490